2021-07-06 17:37:21,057	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 17:37:21,058	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-06 17:47:00,091	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-06 17:47:00,092	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-06 18:30:32,099	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 18:30:32,099	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /owa/auth/logon.aspx
2021-07-06 18:50:37,511	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 18:50:37,512	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-06 19:32:12,764	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-06 19:32:12,764	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-06 20:08:17,359	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 20:08:17,360	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /admin/config.php
2021-07-06 21:48:05,372	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 21:48:05,372	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-06 21:49:44,528	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 21:49:44,529	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-06 21:52:14,503	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-06 21:52:14,503	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 00:36:45,346	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 00:36:45,346	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 00:41:45,456	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 00:41:45,457	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /owa/
2021-07-07 01:02:39,075	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 01:02:39,076	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-07 01:02:40,525	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 01:02:40,526	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 01:03:24,556	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 01:03:24,556	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 01:11:03,688	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 01:11:03,688	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 02:04:06,378	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:06,378	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/jsonws/invoke
2021-07-07 02:04:06,869	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:06,869	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-07 02:04:09,880	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:09,881	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-07 02:04:12,906	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:12,906	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 02:04:13,014	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:13,014	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /index.php
2021-07-07 02:04:15,531	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:15,532	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /wp-content/plugins/wp-file-manager/readme.txt
2021-07-07 02:04:16,882	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:16,882	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /mifs/.;/services/LogService
2021-07-07 02:04:19,831	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:19,831	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /console/
2021-07-07 02:04:20,195	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:20,196	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /Autodiscover/Autodiscover.xml
2021-07-07 02:04:22,069	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:22,070	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 02:04:23,357	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:04:23,357	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-07 02:11:03,425	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:11:03,425	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 02:45:19,049	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 02:45:19,049	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 03:05:19,938	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:05:19,938	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /admin/connection/
2021-07-07 03:05:20,089	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:05:20,090	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /favicon.ico
2021-07-07 03:24:24,243	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:24:24,243	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /admin/connection/
2021-07-07 03:24:24,502	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:24:24,502	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /favicon.ico
2021-07-07 03:33:17,669	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:33:17,670	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /favicon.ico
2021-07-07 03:45:18,675	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:45:18,675	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-07 03:45:19,777	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:45:19,778	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 03:45:22,260	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 03:45:22,260	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 04:57:50,617	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 04:57:50,617	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 06:15:27,299	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:15:27,299	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 06:19:18,291	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:19:18,291	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /ReportServer
2021-07-07 06:20:56,636	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:20:56,636	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 06:30:47,491	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:30:47,491	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 06:30:48,191	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:30:48,191	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 06:46:28,461	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 06:46:28,462	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 07:05:00,021	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:05:00,022	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /login
2021-07-07 07:16:00,322	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:16:00,323	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 07:16:00,965	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:16:00,965	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 07:32:23,736	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:32:23,737	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 07:32:24,495	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:32:24,496	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-07 07:32:25,251	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:32:25,251	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/phpunit.xml
2021-07-07 07:32:26,014	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:32:26,015	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-07 07:40:19,939	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:40:19,940	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-07 07:50:38,678	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:50:38,678	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /ecp/Current/exporttool/microsoft.exchange.ediscovery.exporttool.application
2021-07-07 07:55:38,746	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 07:55:38,746	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 08:09:22,597	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 08:09:22,597	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 09:32:05,294	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 09:32:05,294	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 09:39:31,094	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 09:39:31,094	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 09:43:39,789	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-07 09:43:39,790	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-07 09:52:03,288	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 09:52:03,289	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 10:39:47,502	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:39:47,503	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 10:46:07,626	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:46:07,626	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-07 10:46:08,501	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:46:08,501	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 10:56:58,809	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:56:58,809	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/jsonws/invoke
2021-07-07 10:56:59,916	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:56:59,917	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-07 10:57:00,135	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:00,136	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-07 10:57:01,049	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:01,049	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /solr/admin/info/system
2021-07-07 10:57:01,941	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:01,941	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /console/
2021-07-07 10:57:02,080	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:02,080	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /index.php
2021-07-07 10:57:02,853	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:02,853	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /wp-content/plugins/wp-file-manager/readme.txt
2021-07-07 10:57:03,718	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:03,718	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /Autodiscover/Autodiscover.xml
2021-07-07 10:57:04,567	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:04,568	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-07 10:57:04,628	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:04,629	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 10:57:05,810	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 10:57:05,811	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 12:30:15,412	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 12:30:15,412	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 12:48:31,138	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-07 12:48:31,139	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-07 13:10:24,740	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 13:10:24,740	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 14:01:07,388	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 14:01:07,389	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /actuator/health
2021-07-07 14:16:31,776	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Not Found: /robots.txt
2021-07-07 16:03:32,769	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 16:03:32,770	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 16:22:27,572	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-07 16:22:27,572	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-07 16:38:18,377	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 16:38:18,378	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 16:41:40,672	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 16:41:40,672	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: //a2billing/customer/templates/default/footer.tpl
2021-07-07 17:05:04,023	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 17:05:04,023	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-07 17:05:04,776	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-07 17:05:04,776	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
